sentry_sdk/integrations/aws_lambda.py

```diff
@@ -99,17 +100,22 @@ def sentry_handler(event, context, *args, **kwargs):
                     # Starting the thread to raise timeout warning exception
                     timeout_thread.start()
 
-            try:
-                return handler(event, context, *args, **kwargs)
-            except Exception:
-                exc_info = sys.exc_info()
-                event, hint = event_from_exception(
-                    exc_info,
-                    client_options=client.options,
-                    mechanism={"type": "aws_lambda", "handled": False},
-                )
-                hub.capture_event(event, hint=hint)
-                reraise(*exc_info)
+            headers = _filter_headers(event.get("headers", {}))
```

Comment:
```suggestion
            headers = event.get("headers", {})
```

`_filter_headers` has a different purpose, it is used to remove sensitive data before attaching it to the event. But we don't attach `headers`, we only read one header from it.